microsoft-365/security/office-365-security/manage-tenant-allow-block-list.md

```diff
@@ -99,6 +99,6 @@ By default, allow entries for **domains and email addresses**, **files** and **U
 
 ## What to expect after you add an allow or block entry
 
-After you add an allow entry through the Submissions portal or a block entry in the Tenant Allow/Block List, the entry should start working immediately.
+After you add an allow entry through the Submissions portal or a block entry in the Tenant Allow/Block List, the entry should start working immediately within 99.999% of the time. For the rest it can take upto 24 hours.
```

Comment:
```suggestion
After you add an allow entry through the Submissions portal or a block entry in the Tenant Allow/Block List, the entry should start working immediately 99.999% of the time. For the rest, it could take up to 24 hours.
```